builds/cmake/liblcf-config.cmake.in

```diff
@@ -13,8 +13,16 @@ if(@LCF_SUPPORT_ICU@ EQUAL 1)
 	find_dependency(ICU COMPONENTS i18n uc data REQUIRED)
 endif()
 
-if(@LCF_SUPPORT_XML@)
+if(@LCF_SUPPORT_XML@ EQUAL 1)
 	find_dependency(expat CONFIG REQUIRED)
+elseif(@LCF_SUPPORT_XML@ EQUAL 2)
+	# Fallback to old expat detection
+	find_dependency(EXPAT REQUIRED)
+
+	# Ensure correct target name when the expat config file is used
```

Comment:
Should this be outside of the conditional?

Reply:
That was intentional as I wasn't sure whether ``ALIAS`` works on all versions of CMake. I checked the docs again and ``ALIAS`` is supported since CMake 3.11 so yes I will move it to the outside scope.

(its btw quite dumb that the find module uses EXPAT::EXPAT and the config file expat::expat -_-)